components/shared_data_model/src/shared_data_model/sources/azure_devops.py

```diff
@@ -146,7 +146,7 @@
             metrics=["failed_jobs", "job_runs_within_time_period", "source_up_to_dateness", "unused_jobs"],
         ),
         lookback_days=Days(
-            name="Number of days to look back in selecting job builds to consider",
+            name="Number of days to look back in selecting items to consider",
```

Comment:
"items" is very abstract. Is there really not a term that's better than "job builds" and less abstract than "items"?